File: network/src/main/java/com/pppp/network/poko/Link.java
package com.pppp.network.poko;

import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Root;

@Root
public class Link
{
    @Attribute(name = "rel")
    private String rel;

    @Attribute(name = "type")
    private String type;

    @Attribute(name = "href")
    private String href;

}
			
			

File: network/src/main/java/com/pppp/network/poko/Link.kt
package com.pppp.network.poko

import org.simpleframework.xml.Attribute
import org.simpleframework.xml.Root

@Root
data class Link(
    @field:Attribute(name = "rel", required = false)
    var rel: String? = null,

    @field:Attribute(name = "type", required = false)
    var type: String? = null,

    @field:Attribute(name = "href", required = false)
    var href: String? = null
)
			
			